packages/atomic/package.json

```diff
@@ -61,7 +61,8 @@
     "puppeteer": "5.5.0",
     "rollup": "^2.18.2",
     "rollup-plugin-html": "^0.2.1",
-    "rollup-plugin-typescript2": "^0.27.0"
+    "rollup-plugin-typescript2": "^0.27.0",
+    "stencil": "0.0.5"
```

Comment:
Why is this dependency needed inside atomic?

Reply:
because of that: https://github.com/coveo/ui-kit/pull/573/files/0cfb86451b7faf24a8dae0e63e5c9813b7cb2566#diff-bd0c2b5d4a94884a1504d0cd1be3e064ce154b9d591a3825fcd544626b87f91bR21-R23 
Cannot run if it ain't here.

Reply:
Hmm, clicking the link does not do anything ...

Reply:
Arf. Check the package.json of atomic. The scripts rely on Stencil. Therefore Stencil is needed.

Reply:
> Therefore Stencil is needed.

What do you mean by "Stencil"?

Atomic uses stenciljs to make it easier to build Web Components. This dependency is marked inside the package.json as `@stencil/core`.

The `stencil` dependency listed in the diff is a [different npm package](https://www.npmjs.com/package/stencil) for
"Asynchronous HTML5 templating for Node.js and the browser".

Why is it needed? Which script is not executing without it?

Reply:
So awkward moment :| good catch, I should have seen it...
Well 'fixed' with the big rebase.